src/server/appserver.cpp

```diff
@@ -3943,7 +3943,7 @@ void AppServer::addError(const Error &error) {
                 }
             }
         }
-        for (int errorId: toBeRemovedErrorIds) {
```

Comment:
Sonar usually suggests to use `auto`. It will allow to avoid those cases

Reply:
Done